docs/development/hacking_itk_wasm.md

```diff
@@ -1,69 +1,252 @@
 # Hacking ITK-Wasm
 
-Thank you for contributing a pull request!
+## TLDR
 
-**Welcome to the ITK community!**
+Install Docker, and
+
+```sh
+npm i -g pnpm
+pnpm install
+pnpm build
+pnpm test
+```
+
+and contribute the patch with standard GitHub best practices.
+
+## Introduction
+
+**Welcome to the ITK community! ☀️**
 
 We are glad you are here and appreciate your contribution. Please keep in mind our [community participation guidelines](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CODE_OF_CONDUCT.md).
 
-## Build dependencies
+Thank you in advance for contributing a pull request! 🙏
+
+## High level overview
+
+The following sections describe how to contribute to [ITK-Wasm's constituent parts](../introduction/parts.md).
+
+We follow [standard GitHub contribution best practices]: pull requests made from forks are tested with continuous integration tests, and updates are made via code review before integration.
+
+We use the [conventional commit] standard for our commit message format in the *itk-wasm* repository.
 
-- [Node.js / NPM](https://nodejs.org/en/download/)
-- [Docker](https://docs.docker.com/install/)
-  * On Linux, make sure you can run [`docker` without `sudo`](https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo).
-  * On Windows, we recommend [WSL 2 with Docker enabled](https://docs.docker.com/desktop/windows/wsl/).
-- Bash
+The C++ core and Docker build environment can be developed independently from the CLI, language-specific libraries, and example packages. The latter are developed as a [pnpm workspace]. Individual packages in the workspace can be developed independently after `pnpm install` is executed at the root.
```

Comment:
Are there any special/specific instructions when implementing changes required on both core and individual packages side? For example, I remember `itk-wasm core` used to run the sub-packages by pulling them at run time through `itk-config.ts`. Does this still remain true? How can a developer make local changes to both itk-wasm-core as well as an auxiliary sub-package and link both for testing?

Reply:
Methinks this is addressed in Example packages section. (Just pnpm build at root)

Reply:
Yes, pnpm makes this much better now -- since itk-wasm is linked into the example packages, any changes made there are applied immediately.